docs/DeveloperGuide.md

```diff
@@ -2,24 +2,138 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Add Function
 
+#### System Architecture of the Add Function<br/>
+
+![](team/Architect-digram.PNG)
+
+The Architecture Diagram given above explains the high-level design of the add command.<br/>
+
+Below we will discuss a quick overview of each component.
+
+* UI: The User Interface of the app<br/>
+* Add Command: The main logic command of the add function<br/>
+* Timetable: The arraylist where events that are added are stored accordingly<br/>
+
+#### Design of the Add Function<br/>
+
+UML Class Diagram of the Add function:
+![](team/uml.PNG)<br/>
```

Comment:
The filled in black arrow heads were to be avoided iirc? (inheritance or association issue?)

![image](https://user-images.githubusercontent.com/53657436/97386314-89f98600-190e-11eb-9fde-4327bf33a22c.png)